Not compatible with 6.6.0.0-rc4 bin/build-storefront.js ?

### PHP Version

8.2

### Shopware Version

6.6.0.0-rc4

### Plugin Version

2.0.5

### Actual behaviour

bin/build-storefront.js gives some errors while running:

```
  ERROR in ../../../../../../store.shopware.com/froshlazysizes/src/Resources/app/storefront/src/main.js 1:0-19
  Module not found: Error: Can't resolve 'lazysizes' in '/var/www/html/vendor/store.shopware.com/froshlazysizes/src/Resources/app/storefront/src'

  ERROR in ../../../../../../store.shopware.com/froshlazysizes/src/Resources/app/storefront/src/main.js 2:0-60
  Module not found: Error: Can't resolve 'lazysizes/plugins/native-loading/ls.native-loading' in '/var/www/html/vendor/store.shopware.com/froshlazysizes/src/Resources/app/storefront/src'

  ERROR in ../../../../../../store.shopware.com/froshlazysizes/src/Resources/app/storefront/src/main.js 3:0-52
  Module not found: Error: Can't resolve 'lazysizes/plugins/attrchange/ls.attrchange' in '/var/www/html/vendor/store.shopware.com/froshlazysizes/src/Resources/app/storefront/src'

  ERROR in ../../../../../../store.shopware.com/froshlazysizes/src/Resources/app/storefront/src/main.js 4:0-52
  Module not found: Error: Can't resolve 'lazysizes/plugins/parent-fit/ls.parent-fit' in '/var/www/html/vendor/store.shopware.com/froshlazysizes/src/Resources/app/storefront/src'

  ERROR in ../../../../../../store.shopware.com/froshlazysizes/src/Resources/app/storefront/src/main.js 5:0-52
  Module not found: Error: Can't resolve 'lazysizes/plugins/object-fit/ls.object-fit' in '/var/www/html/vendor/store.shopware.com/froshlazysizes/src/Resources/app/storefront/src'

  frosh-lazy-sizes (webpack 5.89.0) compiled with 5 errors in 814 ms
```

Is it not yet compatible with Webpack5?

### Expected behaviour

Build the storefront js correctly.

### Steps to Reproduce?

Run `bin/build-storefront.sh`

--- a/src/Resources/app/storefront/build/webpack.config.js
+++ b/src/Resources/app/storefront/build/webpack.config.js
@@ -3,7 +3,7 @@ module.exports = function (params) {
     return {
         resolve: {
             modules: [
-                `${params.basePath}Resources/app/storefront/node_modules`
+                `${params.basePath}/Resources/app/storefront/node_modules`
             ]
         }
     };

--- a/src/Storefront/Framework/Twig/Extension/UrlEncodingTwigFilter.php
+++ b/src/Storefront/Framework/Twig/Extension/UrlEncodingTwigFilter.php
@@ -4,7 +4,6 @@ declare(strict_types=1);
 
 namespace Frosh\LazySizes\Storefront\Framework\Twig\Extension;
 
-use Shopware\Core\Content\Media\MediaEntity;
 use Symfony\Component\DependencyInjection\Attribute\AutoconfigureTag;
 use Twig\Extension\AbstractExtension;
 use Twig\TwigFilter;

